ui/admin/app/components/breadcrumbs/item/index.hbs

```diff
@@ -0,0 +1,16 @@
+{{#each this.breadcrumbsService.containers as |container|}}
```

Comment:
When we would have more than one breadcrumb container?

Reply:
oop nope, ill refactor this as well, thank you!

Reply:
Hmm I wasn't exactly saying this needs to be refactored but I was wondering when one would have more than one container, what's the use case since it looks like the service supports more than one container and multiple ones can be registered

Reply:
The use case I can think of for having multiple containers is for rendering breadcrumbs with different attributes applied to diff containers. This won't apply to our use case tho. Since the container we are registering is always going to be the` <ol>` element of the `<Hds::Breadcrumb/>` component. So we will at most have 1 container registered at all times.

Reply:
Yup or if for some reason you needed multiple sets of breadcrumbs on the same page. I don't think we'd ever have multiple containers and it seems like the implementation makes that assumption [here](https://github.com/hashicorp/boundary-ui/blob/6116b9aba8a05762deb682a1354cda3e09fb18d9/ui/admin/app/components/breadcrumbs/item/index.js#L17) anyways and would always grab the first and only container